### PR TITLE
Introduce disk top IO

### PR DESCRIPTION
**Pull request by vadmeste**
This shows IO stats of MinIO backend disks

Depends on https://github.com/minio/madmin-go/pull/111

### CODE REVIEW

**Comment by harshavardhana** [comment]
How does it look do you have an asciinema?

File: cmd/admin-top-disk.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+	cli.IntFlag{
+		Name:  "count, c",
+		Usage: "show the maximum top disks",
+		Value: 10,
+	},
+}
+
+var adminTopDiskCmd = cli.Command{
+	Name:            "disk",
+	Usage:           "Show disk stats in real-time",
```
Comment (harshavardhana) [inline cmd/admin-top-disk.go:42]:
```suggestion
	Usage:           "show current disk statistics",
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Comment by vadmeste** [comment]
> How does it look do you have an asciinema?

https://asciinema.org/a/hDH7gZXzYK5Q9BIuvK0XlVUeW

I actually failed to find 4 different disks. Overall there is sorting by column, disk usage, tps, read/write speed, await and utilisation.

There is also left right arrow to change the current pool

**Comment by harshavardhana** [comment]
> I actually failed to find 4 different disks. Overall there is sorting by column, disk usage, tps, read/write speed, await and utilisation.

Well you can do `losetup`  @vadmeste

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**